### PR TITLE
Add Yaml Documentation

### LINKED ISSUE #1613: How to generate API documentation in yaml swagger format?
In Nelmio UI there is missing some yaml preview of generated API, is it possible to generate documentation in yaml format? Some tools for testing like soap ui are able to import yaml swagger file and then soap ui is able to work with this file and apis. Solution?

### PR DESCRIPTION
**Pull request by saleh199**
Fixes #1613

### CODE REVIEW

File: Controller/JsonDocumentationController.php [hunk 1]
```diff
@@ -18,7 +18,7 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
 
-final class DocumentationController
+final class JsonDocumentationController
```
Comment (GuilhemN) [inline Controller/JsonDocumentationController.php:21]: I'd rather avoid this renaming, it is a backward compatibility break that is not necessary and if you really want to do it, I'd rather have it in a separate PR with backward compatibility layers.

**Review by GuilhemN — COMMENTED** [review]
Thank you for this PR!
I think we should avoid the renaming of the existing controller but except for this, it's a great addition :)

File: Controller/JsonDocumentationController.php (hunk 1 above)
Comment (saleh199) [reply Controller/JsonDocumentationController.php]: Yes, you are right, I already reverted the change of the controller

**Comment by GuilhemN** [comment]
Thank you very much @saleh199!

**Pull request merged**